support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render } from '@testing-library/react';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import type { ActiveProductKey } from 'helpers/productCatalog';
+import { PaymentTsAndCs } from './paymentTsAndCs';
+
+// Mocking price retrieval from productCatalog (not available in window at runtime)
+jest.mock('helpers/supporterPlus/benefitsThreshold', () => ({
+	getLowerProductBenefitThreshold: () => 12,
+}));
+// jest.mock('./TsAndCsRenewal', () => ({
```

Comment:
Do you need this commented out mock for `TsAndCsRenewal` any more? Can it be removed?

Reply:
All mock calls have been removed now in favour of passing in the threshold amount (ie £12 currently for GBP) into the `paymentTsAndCs` component as a prop. Its now calculated higher up in the `checkoutComponent` (as its uses the backend for this info - which the tests cannot do) and is passed down as prop `thresholdAmount` to the `paymentTsAndCs` component. Unfortunately `thresholdAmount` is the S+ minimum level used in its Ts&Cs copy which think we need but perhaps there is a better name?

Reply:
OK thanks for explaining @paul-daniel-dempsey. My comment was referring to the commented out code on line 10-12 and whether that could be removed?

Reply:
Re: `thresholdAmount` is this just the Supporter Plus price from the product catalog? @paul-daniel-dempsey

Reply:
re `thresholdPrice`, it is the current S+ price from productCatalog.  This is not a static number however, its loaded onto the window Guardian object I believe through a `ProductPricesSchema` (through Play controllers I believe), thus not available to the test components so passed in as a prop.

Reply:
> Perhaps rename to `supporterPlusPrice` might be clearer?

Yeah I think this is a more suitable name 👍, 'threshold' feels more pre-Tiers nomenclature.

Reply:
Have renamed this now. Think a further refactor to remove `threshold` in all functions throughout needs to come but will put that in the followup tidy-up TsAndCs PR. Hope thats ok.

Reply:
Will resolve this for now, there is a comment [here](https://github.com/guardian/support-frontend/pull/6848/files#r2003732943) explaining what we ideally would want